src/main/java/com/smashingmods/reroll/config/Config.java

```diff
@@ -37,6 +44,15 @@ public static void readConfig() {
         }
     }
 
+    public static Block getSpawnBlock() {
+        if(spawnBlock == null) {
+            if(!blockSpawnName.equals("null") && !blockSpawnName.equals("")) {
+                spawnBlock = ForgeRegistries.BLOCKS.getValue(new ResourceLocation(blockSpawnName));
+            }
+        }
+        return spawnBlock;
+    }
```

Comment:
Please extract logic from the Config file and put it in the relevant method where this logic is being used. For example, spawnblockname can be a registry name and then when you need to use it, you instantiate a new block using the name.